docs/source/tutorials/staging_data.ipynb

```diff
@@ -1,20 +1,340 @@
 {
```

Comment:
<p><code>Flags are set automatically</code> - is that always the case?  Either way, the user can also specify flags. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>yes, <code>CREATE_PARENTS</code> is a default flag and  <code>RECURSIVE</code> is set by RP if input is a dictionary, but I've also added that user can set flags explicitly </p>